config/myproxy-server.config

```diff
@@ -0,0 +1,21 @@
+        authorized_retrievers      "*"
+        default_retrievers         "*"
+        authorized_renewers        "*"
+        authorized_key_retrievers  "*"
+        trusted_retrievers         "*"
+        default_trusted_retrievers "none"
+        max_cert_lifetime          "72"
+        disable_usage_stats        "true"
+        cert_dir                   "/etc/grid-security/certificates"
+
+        pam_id "myproxy"
+        pam "required"
+
+        certificate_issuer_cert "/var/lib/globus-connect-server/myproxy-ca/cacert.pem"
+        certificate_issuer_key "/var/lib/globus-connect-server/myproxy-ca/private/cakey.pem"
+        certificate_issuer_key_passphrase "globus"
+        certificate_serialfile "/var/lib/globus-connect-server/myproxy-ca/serial"
+        certificate_out_dir "/var/lib/globus-connect-server/myproxy-ca/newcerts"
+        certificate_issuer_subca_certfile "/var/lib/globus-connect-server/myproxy-ca/cacert.pem"
+        certificate_mapapp /esg/config/myproxy/myproxy-certificate-mapapp
```

Comment:
Do the `certificate_mapapp` and `certificate_extapp` need quotes. It looks like all the others have them

Reply:
The values for those two keys were unquoted in the classic installer.  I'll look at the myproxy documentation and do a manual test to see if quoting matters on those. But I left them unquoted as it seemed to work in the classic installer.

Reply:
Sounds good